### PR TITLE
Second milestone

### PR DESCRIPTION
**Pull request by srizzuto**
In this second part of the project, we are adding sections to the middle section of the website while caring about responsiveness and the use of bootstrap.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,94 +18,1111 @@
...
-                </path>
-              </g>
-            </svg>
-          </a>
+<body class="mx-auto mwidth d-block">
+  <div class="container-fluid bg-white mwidth">
+    <header>
+      <div class="container-fluid empty-container"></div>
```
Comment (sinansevgi) [inline index.html]: [Optional] Why do you have this empty container? It creates white empty space on top of your page.

**Review by sinansevgi — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️ 

------------------------

Hello Team,

Keep up the great work! Your project looks amazing. 

There is one small issue with the order of elements on mobile resolution.  

---------------------------

- [x] On mobile resolution order of elements doesn't match with the original site. Please check the below table and fix the order on mobile resolution. (Tip: You can use [order classes](https://getbootstrap.com/docs/4.0/layout/grid/#order-classes))

Correct order for mobile:

<table>
    <thead>
        <tr>
            <th>Mobile</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>TOP STORY</th>
        </tr>
        <tr>
            <td>FEATURED STORIES</th>
        </tr>
        <tr>
            <td>THE DEBATE</th>
        </tr>
        <tr>
            <td>OPINION</th>
        </tr>
        <tr>
            <td>SPONSORED INSIGHT</th>
        </tr>
        <tr>
            <td>MORE STORIES</th>
        </tr>
        <tr>
            <td>LATEST NEWS</th>
        </tr>
        <tr>
            <td>MY TURN</th>
        </tr>
        <tr>
            <td>CULTURE & TRAVEL</th>
        </tr>
        <tr>
            <td>SPONSORED INSIGHT</th>
        </tr>
        <tr>
            <td>FREE SIGN UP</th>
        </tr>
        <tr>
            <td>SPONSORED INSIGHT</th>
        </tr>
    </tbody>
</table>

Please be sure you didn't change the order of elements in tablet and desktop resolution while fixing mobile :warning: 


-----

## "As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 code reviews left for this project."

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi

File: index.html (hunk 1 above)
Comment (srizzuto) [reply index.html]: Hello! Thanks for your quick reply.

The empty container was emulating the one on the original page. I think it has some back-end proposes but I just created it anyways. I can remove it if it represents an issue!

Comment (sinansevgi) [reply index.html]: I think you may get confused by the space added by archive.org. If you check [the original page](https://www.newsweek.com/), it doesn't exist. This is not an important issue.  I forgot to mark this as optional. Sorry :smile:  You can ignore this comment, and focus on ordering elements if you want. But your project will look better if you remove it.

**Comment by srizzuto** [comment]
Changes made as required!

**Review by descholar-ceo — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
## :clap: :clap: :clap: Congratulations @srizzuto and @vikitaotiz  , your project is approved :tada: :tada: :tada: 
You have implemented all requested changes from the previous review, but I noticed one critical issue which might affect your project in the future, maybe it's a bug which is introduced while working on the requested changes, that issue is that the `Opinion` section is missing on mobile screens, I highly recommend you add it before merging the changes on the default branch

Don't forget to fill the mandatory project completion form, and see you to next project,

Happy coding :computer: :computer: :computer: 

Re-reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Pull request merged**